taipy/core/data/data_node.py

```diff
@@ -29,12 +30,35 @@
 from ..common._warnings import _warn_deprecated
 from ..exceptions.exceptions import DataNodeIsBeingEdited, NoData
 from ..job.job_id import JobId
+from ..notification.dn_scenario_cache import SubmittableStatusCache
 from ..notification.event import Event, EventEntityType, EventOperation, _make_event
 from ._filter import _FilterDataNode
 from .data_node_id import DataNodeId, Edit
 from .operator import JoinOperator
 
 
+def _update_parent_submittable_cache_on_edit_in_progress(fct):
+    @functools.wraps(fct)
+    def _do_update_parent_submittable_cache_on_edit_in_progress(dn, *args, **kwargs):
+        fct(dn, *args, **kwargs)
+        if not dn._edit_in_progress:
+            SubmittableStatusCache.remove(dn.id)
```

Comment:
Same here, we don't know if we should remove the dn from the cache or not. It depends on the value of `_last_edit_date`.